Documentation/Storage-Configuration/NFS/nfs.md

```diff
@@ -88,6 +88,15 @@ can be used. This creates an export for the `/test` pseudo path.
 ```console
 ceph nfs export create cephfs my-nfs /test myfs
 ```
+!!! warning
+    RGW NFS export is experimental for the moment
+
+For creating an NFS export for the CephNFS and RGW(CephObjectStore) example manifests, the below command
+can be used. This creates an export for the `/testrgw` pseudo path on an existing bucket bkt4exp.
+
+```console
+ceph nfs export create rgw my-nfs /testrgw bkt4exp
+```
```

Comment:
Thanks for the doc update. Since this is an experimental feature, let's move this to the `nfs-advanced.md` doc. And please create a new section for it as well.

Reply:
there you go

Reply:
@BlaineEXE any slot to proceed on this?